akka-projection-r2dbc/src/main/resources/reference.conf

```diff
@@ -27,12 +27,27 @@ akka.projection.r2dbc {
     # It should not be larger than the akka.projection.r2dbc.offset-store.time-window.
     backtracking-window = ${akka.persistence.r2dbc.query.backtracking.window}
 
-    # Remove old entries when older than this duration from the offset store database.
-    delete-after = 1 day
-
-    # Remove old entries when older than delete-after from the offset store database
-    # with this frequency. Can be disabled with `off`, `none`, or empty ("").
-    delete-interval = 10 minutes
+    # Delete old entries from the offset store database when offset timestamps
+    # are older than this duration.
+    # The deletion criteria is combined with delete-after-consumed.
+    # Delete when offset timestamp older than delete-after and consumed timestamp
+    # older than delete-after-consumed.
+    delete-after = 10 days
```

Comment:
This might be an incompatible change, which is not possible. The system was previously running with the default of 1 day, and then this is changed to 10 days. For a new event where the offset was deleted 5 days ago by the previous version (1 day retention) it will trigger "Rejecting unknown" and replay events for that entity from seqNr 1.